openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Tag.spec.ts

```diff
@@ -360,3 +363,107 @@ test.describe('Tag Page with Limited EditTag Permission', () => {
     }
   });
 });
+
+test.describe('Create tag with domain', () => {
```

Comment:
would recommend to move this under `Tag Page with Admin Roles` describe.
Also, move logic to tag.ts, refer above tests